src/generic/stepper/Header.jsx

```diff
@@ -5,13 +5,12 @@ import Icon from './Icon';
 import Line from './Line';
 import { StepperContext } from './StepperContext';
 
-export default function Header({ steps, className }) {
+export default function Header({ steps, className, isFirstStep }) {
```

Comment:
rather than passing this `istFristStep` here, what you can also do is

move the following code into utils.js and use that on both pages, in that case, you do not have to pass the param along. 
```
import {  useLocation } from 'react-router';

const { pathname } = useLocation();
const { path: pagesAndResourcesPath } = useContext(PagesAndResourcesContext);
const discussionsPath = `${pagesAndResourcesPath}/discussions`;

return pathname === discussionsPath;
```

Reply:
done